frontend/src/app/components/containers/complaints/outcomes/hwcr-complaint-assessment.tsx

```diff
@@ -112,10 +112,10 @@ export const HWCRComplaintAssessment: FC = () => {
     if (complaintData) {
       const officer = getSelectedOfficer(assignableOfficers, personGuid, complaintData);
       setSelectedOfficer(officer);
-      dispatch(getAssessment(complaintData.id, officerList ?? undefined));
+      dispatch(getAssessment(complaintData.id));
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [complaintData]);
+  }, [complaintData, officers]);
```

Comment:
this dependancy array should include all of the objects you're using in the useEffect, but again the officers object din't being used here and is redundant. 


```
//-- suggested dependancy array

useEffect(() => {
  //-- existing code
  // eslint-disable-next-line react-hooks/exhaustive-deps
}, [assignableOfficers, complaintData, personGuid]);